fast-enrichment-service/src/utils/xmlParser.ts

```
@@ -0,0 +1,145 @@
+    if (!obj || typeof obj !== 'object') {
+      return null;
+    }
+
+    // Check if this object contains CdtrAcct structure
+    if (obj.CdtrAcct) {
+      return this.extractAccountId(obj.CdtrAcct);
+    }
+
+    // Recursively search through the object
+    for (const key in obj) {
+      if (obj.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)